### DISCUSSION TITLE
Independent shooter side speed control vs distance

### DISCUSSION THREAD

**Discussion by PChild** [team401/2024-Robot-Code#80] [General]
Related to #78 how do we want to handle independent left / right speeds at different distances? Just interpolating doubles for both sides? Some fixed proportion between the two sides? Some fixed RPM offset?

**Comment by jkleiber** [comment]
The baseline path forward right now is that we offset the RPM by some fixed proportion.

If we determine in testing that the note flies better with a variable proportion/offset, we will be forced to maintain either 2 lookup tables, or a single lookup table with both RPM values as dependent variables on distance from goal

**Comment by honzikschenk** [comment]
Apparently Mechanical uses a fixed position so I went with that approach for the time being.
